### PR TITLE
DOC: Add docs for __call__ on _StepScan

### PR DESCRIPTION
**Pull request by ericdill**

### CODE REVIEW

**Comment by ericdill** [comment]
dscan? now shows helpful info
```
In [6]: dscan?
Type:           DeltaScan
String form:    <bluesky.simple_scans.DeltaScan object at 0x104c4b320>
File:           ~/dev/python/bluesky/bluesky/simple_scans.py
Signature:      dscan(self, motor, start, finish, intervals, time=None, subs=None, **kwargs)
Docstring:      dscan (also known as lup)
Call signature: dscan(motor, start, finish, intervals, time=None, subs=None, **kwargs)
Call docstring:
Invoke the scan

Parameters
----------
motor
start : number
    The start point of the motion
finish : number
    The finish point of the motion
intervals : int
    The number of steps between `start` and `finish`
time : number
    The acquire time of the detector(s)?
subs : dict
    The temporary subscriptions to add to **this scan only**. These
    subscriptions are **not** persistent
```

**Comment by ericdill** [comment]
Whereas previously:
```
In [3]: dscan?
Type:           DeltaScan
String form:    <bluesky.simple_scans.DeltaScan object at 0x104a1ad30>
File:           ~/dev/python/bluesky/bluesky/simple_scans.py
Signature:      dscan(self, motor, start, finish, intervals, time=None, subs=None, **kwargs)
Docstring:      dscan (also known as lup)
Call signature: dscan(motor, start, finish, intervals, time=None, subs=None, **kwargs)
```

File: bluesky/simple_scans.py [hunk 1]
```diff
@@ -228,6 +228,23 @@ class _StepScan(_BundledScan):
...
+        ----------
+        motor
+        start : number
+            The start point of the motion
+        finish : number
+            The finish point of the motion
+        intervals : int
+            The number of steps between `start` and `finish`
+        time : number
+            The acquire time of the detector(s)?
```
Comment (ericdill) [inline bluesky/simple_scans.py]: I'm not sure exactly how this works. Does this set **all** the acquire times of the detectors?

Comment (tacaswell) [inline bluesky/simple_scans.py]: It is set through the magical DWIM interface which seems to do what _noone_ wants.  For AD it sets the total acquisition time to this value by changing the exposure time (iirc) but for scalers just sets the count time.

**Comment by danielballan** [comment]
Good. Thanks for adding this.

**Comment by ericdill** [comment]
Any blocking issues on this being merged?

**Comment by tacaswell** [comment]
kicked to restart tests

**Comment by ericdill** [comment]
It is literally only a documentation change

File: bluesky/simple_scans.py (hunk 1 above)
Comment (danielballan) [inline bluesky/simple_scans.py]: @ericdill I assumed you wanted to remove the '?' or elaborate before merging.

Comment (danielballan) [inline bluesky/simple_scans.py]: (This is due for a refactor once ophyd is ready to re-deploy though, so we can just fix it at that time.)

**Pull request merged**